subprojects/launcher/src/test/groovy/org/gradle/launcher/MainTest.groovy

```diff
@@ -48,7 +48,7 @@ class MainTest extends Specification {
             [complete: { completedSuccessfully = true }, completeWithFailure: { completedWithFailure = true; failure = it }] as ExecutionCompleter
         }
 
-        protected CommandLineActionFactory createActionFactory() {
+        CommandLineActionFactory createActionFactory() {
```

Comment:
Why this change?

Reply:
it overrides a public method

Reply:
Is this something you discovered accidentally, or something that Groovy 2.5.7 now fails because of?

Reply:
It's a failure now